cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js

```diff
@@ -60,6 +60,7 @@ export default function FileTable({ items }) {
                   },
                 }
               : { href: getFileUri(file.key) };
+          const label= file.type=='index.txt' ? 'Download index' : 'Download';
```

Comment:
could that be also `index.json`?

Reply:
THe index.json entries seem to be filtered out at some point (I haven't figure it out where exactly).  See for instance [this entry](https://opendata.cern/record/28003), which contains a normal file and a  file set:
![Screenshot 2024-03-27 at 16 27 14](https://github.com/cernopendata/opendata.cern.ch/assets/12909623/fe3cc05b-1119-47cc-8bc5-862605af8814)

If we look at the entry on opensearch, it has three files:
![Screenshot 2024-03-27 at 16 28 00](https://github.com/cernopendata/opendata.cern.ch/assets/12909623/9c3b859f-e3a4-4215-9b09-630908999ffd)